src/main/java/ui/components/NavigableListView.java

```diff
@@ -111,25 +114,41 @@ public void restoreSelection() {
                 selectedIndex = Optional.of(nextIndex);
 
                 // The next item will be considered selected
-                onItemSelected.accept(nextIndex);
+                triggerItemSelected(nextIndex);
             }
         }
     }
 
+    /**
+     * Triggers the selected item on the card at the specified index for actions that are not right clicks
+     */
+    private void triggerItemSelected(int index){
+        onItemSelected.accept(index, !IS_RIGHT_CLICK);
+    }
+
+    /**
+     * Triggers the selected item on the card at the specified {index} for actions that are right clicks
```

Comment:
Oh yup, that's what I was looking for! Thanks! But researched this further and realized it's only needed for code blocks to be used in javadoc comments above methods, which is not applicable in my case.